src/manifest.c

```diff
@@ -1463,7 +1463,6 @@ int man_write_fw_ace_v1_5(struct image *image)
 	m->desc.header.build_version = image->fw_ver_build;
 
 	m->desc.header.feature_mask = 0x2; // -> should be feature mask - to fix
-	m->desc.header.fw_image_flags = 0x2; // -> should be feature mask - to fix
```

Comment:
previous implementation have issues? if not, change 0x2 --> 0x12 will make changes more less.

Reply:
it is fixed and it should be built from .toml file